packages/flutter/lib/src/widgets/focus.dart

```diff
@@ -79,8 +96,15 @@ class Focus extends StatefulComponent {
 
   final Widget child;
 
+  /// The key that currently has focus globally in the entire focus tree.
+  ///
+  /// This field is always null when asserts are disabled.
```

Comment:
we usually say something like "This field is always null except in checked mode" or similar, as opposed to talking about disabling asserts.